mlir/lib/Dialect/LLVMIR/IR/NVVMDialect.cpp

```diff
@@ -506,7 +507,7 @@ LogicalResult MmaOp::verify() {
       expectedA.emplace_back(1, f64Ty);
       expectedB.emplace_back(1, f64Ty);
       expectedC.emplace_back(2, f64Ty);
-      // expectedC.emplace_back(1, LLVM::getFixedVectorType(f64Ty, 2));
+      // expectedC.emplace_back(1, VectorType::get(2, f64Ty));
```

Comment:
I wonder if we should just remove this commented out line? Given that it has no comment, it's probably safe to assume that this is indeed just a remnant.

Reply:
Will send a separate PR to remove this. (And send it to the original author.)